### DISCUSSION TITLE
any way to set remote bag source in web url?

### DISCUSSION THREAD

**Discussion by lanyusea** [foxglove/community#78] [Q&A] [answered]
in webviz I can open a remote bag by adding `?remote-bag-url=` following the url

but I cannot find any similar functionality in foxglove document

is there any similar way I can achieve this in foxglove?

**Comment by amacneil** [comment]
Yes! It is coming very soon:

https://github.com/foxglove/studio/issues/2025
https://github.com/foxglove/studio/pull/2171

cc @foxymiles

**Comment by defunctzombie** [comment, marked as answer]
This feature is now available. Here's an example url for remote bagfiles:

https://studio.foxglove.dev/?ds=ros1-remote-bagfile&ds.url=https://storage.googleapis.com/foxglove-public-assets/demo.bag

We're still working on the docs for this feature - and stabilizing the url query arguments so feedback is welcome!
